docs/guides/genomics-platform/analyzing-data/rnaseq-expression-classification.md

```diff
@@ -114,39 +169,48 @@ Once the resulting analysis job completes, an HTML plot of the results should be
 
```

Comment:
Can't see it on the diff, but `## Cost Estimation` should become `### Cost Estimation` and we should consider moving it to the last section in Overview.